java/Ice/greeter/README.md

```diff
@@ -0,0 +1,43 @@
+# Greeter
+
+The Greeter demo illustrates how to send a request and wait for the response.
+
+This demo provides two implementations for the server:
+- `server`: A synchronous dispatch implementation
+- `serveramd`: An asynchronous dispatch implementation using Ice's Asynchronous Method Dispatch (AMD)
+
+The same client works with both.
+
+## Building the Project
+
+This project uses Gradle and the [Application plugin]. To build the client and server applications, run:
```

Comment:
I assume this is a reasonably common thing to do?

Reply:
That is what `gradle init` does when you create a new Java Application project.